plugins/psd-claude-coding-system/agents/llm-specialist.md

```
@@ -120,6 +120,96 @@ function buildPrompt(template: string, variables: Record<string, any>) {
 ```
 
+## Security Considerations
+
+**CRITICAL**: LLM integrations introduce unique security risks. Address these before deployment.
+
+### Prompt Injection Prevention (CWE-94)
+
+Prompt injection is when malicious user input manipulates LLM behavior. Prevent it:
+
+- **Never directly concatenate user input** into prompts:
+  ```typescript
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)